### PR TITLE
IDE-2760 update test file to make it run ok

### PR DESCRIPTION
**Pull request by rui123wang**
IDE-2760 update test file to make it run ok

### CODE REVIEW

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/SwtbotBase.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public static void beforeClass() throws Exception {
 
 		SWTBotPreferences.TIMEOUT = 30 * 1000;
 
-		System.setProperty(SWTBotPreferenceConstants.KEY_TIMEOUT, "30000");
```
Comment (yanan) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/SwtbotBase.java]: Why should we change this?

File: ui-tests/tools/com.liferay.ide.ui.server.tests/src/com/liferay/ide/ui/server/tests/ServerTomcatTests.java [hunk 2]
```diff
@@ -36,57 +36,57 @@
 	@BeforeClass
 	public static void prepareServer() throws IOException {
 		envAction.unzipServer();
-
+
```
Comment (yanan) [inline ui-tests/tools/com.liferay.ide.ui.server.tests/src/com/liferay/ide/ui/server/tests/ServerTomcatTests.java]: Did you run command about formate for this file?

File: ui-tests/tools/com.liferay.ide.ui.server.tests/src/com/liferay/ide/ui/server/tests/ServerTomcatTests.java [hunk 3]
```diff
@@ -36,57 +36,57 @@
...
+	
 		viewAction.showServersView();
-
+	
 		String serverStoppedLabel = serverName + "  [Stopped]";
-
+	
 		viewAction.serverStart(serverStoppedLabel);
-
-		sleep(200000);
```
Comment (yanan) [inline ui-tests/tools/com.liferay.ide.ui.server.tests/src/com/liferay/ide/ui/server/tests/ServerTomcatTests.java]: Good job catching this!

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/SwtbotBase.java (hunk 1 above)
Comment (rui123wang) [reply ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/SwtbotBase.java]: When running, the current state timeout is displayed

**Pull request closed without merging**